week3/solutions/how_many.py

```diff
@@ -0,0 +1,19 @@
+def how_many(dictionary):
+  '''
+  aDict: A dictionary, where all the values are lists.
+
+  returns: int, how many values are in the dictionary.
+  '''
+  how_many = 0
+  for key in dictionary:
+      how_many += len(dictionary[key])
+  return how_many
```

Comment:
How can you rewrite this?

Reply:
```
def how_many(dictionary):
  return sum([len(dictionary[key]) for key in dictionary])
```

Is this better/what you were thinking? I like these one line functions :)